src/olympia/editors/tests/test_helpers.py

```diff
@@ -137,14 +137,30 @@ def test_last_review(self):
         doc = pq(self.table.render_last_review(row))
         assert doc.text() == u'0.34.3b on 2016-01-01'
 
-    def test_authors(self):
+    def test_authors_few(self):
         row = Mock()
-        row.authors = [(123, 'bob'), (345, 'steve')]
+        row.authors = [(123, 'bob'), (456, 'steve')]
         doc = pq(self.table.render_authors(row))
-        url = UserProfile.create_user_url(123, username='bob')
-        assert doc('span').text() == 'bob ...'
-        assert doc('span a').attr('href') == url
-        assert doc('span').attr('title') == 'bob steve '
+        assert doc('span').text() == 'bob steve'
+        assert doc('span a:eq(0)').attr('href') == UserProfile.create_user_url(
+            123, username='bob')
+        assert doc('span a:eq(1)').attr('href') == UserProfile.create_user_url(
+            456, username='steve')
+        assert doc('span').attr('title') == 'bob steve'
+
+    def test_authors_four(self):
```

Comment:
Oh yeah, it wouldn't belong here. I could see setting up the data to test it in the models being tricky.